052-connector-state.md

```diff
@@ -0,0 +1,55 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Support stopping connectors
+
+This proposal aims at adding support for stopping Kafka Connect connectors.
```

Comment:
To join the dots, can you link to the KIP where this was discussed in Apache Kafka?

Reply:
Added a link to KIP-875.